src/components/FeaturesDemo/index.tsx

```diff
@@ -0,0 +1,726 @@
+            store.setWindow(
+                1,
+                {
+                    pageData: {
+                        type: PageTypes.App,
+                        props: {
+                            products: PRODUCTS_SAMPLE_DATA,
+                            searchText: "",
+                            cartCount: 0,
+                        },
+                    },
+                    props: {
+                        isPageLoading: false,
+                    },
+                },
+                signal,
+            );
+            await pause(PAUSE_MD, signal);
+        },
+        async function typeWrongSetTextCommand() {
+            const codeLineData = getById(codeRefs, CodeId.FindInput);
+            scrollToLineIfNeeded(codeLineData.ref, codeElementRef);
+            await typeCode(codeLineData, codeCleanupRef);
+
+            await typeConsoleMessage(
+                getById(consoleRefs, ConsoleId.SendHelloSearch),
+                consoleCleanupRef,
+            );
+            await pause(PAUSE_SM, signal);
+
+            await typeConsoleMessage(
+                getById(consoleRefs, ConsoleId.HelloSearchFailure),
+                consoleCleanupRef,
+            );
+            await pause(PAUSE_MD, signal);
+        },
+        async function typeCorrectSetTextCommand() {
+            await typeCode(
+                { ...getById(codeRefs, CodeId.FindInput), messageIndex: 1 },
+                codeCleanupRef,
+            );
+
+            await typeConsoleMessage(
+                getById(consoleRefs, ConsoleId.SendInputSearch),
+                consoleCleanupRef,
+            );
+            await pause(PAUSE_SM, signal);
+
+            await typeConsoleMessage(
+                getById(consoleRefs, ConsoleId.InputSearchSuccess),
+                consoleCleanupRef,
+            );
+            await pause(PAUSE_MD, signal);
+        },
+        async function scrollToCommand() {
+            const codeLineData = getById(codeRefs, CodeId.ScrollToInput);
+            scrollToLineIfNeeded(codeLineData.ref, codeElementRef);
+            await typeCode(codeLineData, codeCleanupRef);
+
+            await typeConsoleMessage(
+                getById(consoleRefs, ConsoleId.SendScrollIntoView),
+                consoleCleanupRef,
+            );
+            await pause(PAUSE_SM, signal);
+
+            await typeConsoleMessage(
+                getById(consoleRefs, ConsoleId.ScrollIntoViewSuccess),
+                consoleCleanupRef,
+            );
+
+            // eslint-disable-next-line @typescript-eslint/non-nullable-type-assertion-style
+            const searchInput = appBrowserPageRef.current
+                ?.getUnsafePageRef()
+                ?.current?.querySelector('input[data-role-id="search-input"]') as HTMLInputElement;
+            const offset = searchInput.offsetTop;
+            appBrowserPageRef.current?.setScrollTop(
+                offset - 36 - 40 /* height of search bar, height of header */,
```

Comment:
Can we use some constant values here for 36 and 40 and use them in css styles?

Reply:
replaced with constants.